etesync_server/settings.py

```diff
@@ -135,3 +135,13 @@
 
 # Cors
 CORS_ORIGIN_ALLOW_ALL = True
+
+# Modify PYTHONPATH to add an etesync_site_settings module for importing to
+# override settings in this file.
```

Comment:
You don't even have to modify python path if you just load it from the base source directory. This is actually how the etesync production server does it.

Reply:
My container deployments are read-only filesystems and the file needs written so it lives over in `/run` rather than the top-level directory, but I'll remove the comment.